cluster/manifests/prometheus/ingress.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
   hosts:
     - system-prometheus.{{ .Values.hosted_zone }}
+    - system-prometheus-{{ .Cluster.Alias }}.{{ .Values.hosted_zone }}
```

Comment:
a bit ugly name TBH

Reply:
Do we plan to run more than one cluster in each account in the future?

Reply:
Talked with @mikkeloscar about it and if this is the best way to solve it go for it, but I would assume we could also check @AlexanderYastrebov 's jwtMetrics reports to figure out who is using these targets before deploying this change to all clusters.

Reply:
jwtMetrics won't help (we can why separately) but we can enable 2xx access log and it will contain token uid (token is present and required for both services and oauthGrant routes)

Reply:
I have added `enableAccessLog()` now.